qt/widgets/common/inc/MantidQtWidgets/Common/FunctionModelDataset.h

```diff
@@ -0,0 +1,34 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "DllOption.h"
+#include "MantidQtWidgets/Common/Spectra.h"
+
+#include <QList>
+#include <QString>
+#include <QStringList>
+
+namespace MantidQt {
+namespace MantidWidgets {
+
+struct EXPORT_OPT_MANTIDQT_COMMON Dataset {
```

Comment:
Thanks for putting in the changes in around this. Can I ask we rename this as `Dataset` is a bit of a overloaded term w.r.t to mantid and other things like [scipp](https://scipp.github.io/).

I noticed the header is called `FunctionModelDataset` - can we use this name?

Reply:
Can we also add some class documentation just describing the intention of the type?